src/test/java/eu/bbmri/eric/csit/service/negotiator/integration/service/NegotiationLifecycleServiceImplTest.java

```diff
@@ -172,6 +175,17 @@ void sendEventForResource_approvedNegotiation_Ok() throws IOException {
         NegotiationResourceState.REPRESENTATIVE_CONTACTED,
         resourceLifecycleService.sendEvent(
             negotiationDTO.getId(), "biobank:1:collection:2", NegotiationResourceEvent.CONTACT));
+    // check that the negotition has a resource record with the assigned state
```

Comment:
please remove comment and replace with appropriately named method that experesses the logic in its name

Reply:
Fixed in 673d2d0e5aa92b06fda147cf34949dd7444d4de4